lib/aruba/api.rb

```diff
@@ -98,12 +98,54 @@ def directory?(file)
       File.directory? expand_path(file)
     end
 
+    # Check if path is absolute
+    #
+    # @return [TrueClass, FalseClass]
+    #   Result of check
+    def absolute?(path)
+      Pathname.new(path).absolute?
+    end
+
+    # Check if path is relative
+    #
+    # @return [TrueClass, FalseClass]
+    #   Result of check
+    def relative?(path)
+      Pathname.new(path).relative?
+    end
+
     # Return all existing paths (directories, files) in current dir
     #
     # @return [Array]
     #   List of files and directories
     def all_paths
```

Comment:
Besides that, I would like add aruba-helpers for most/all methods of http://ruby-doc.org/core-2.2.0/FileTest.html.